devise.rb

```diff
@@ -43,6 +43,11 @@
   gsub_file('app/views/layouts/application.html.erb', "<%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>", scripts)
 end
 gsub_file('app/views/layouts/application.html.erb', "<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>", "<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>")
+style = <<~HTML
+  <meta name=“viewport” content=“width=device-width, initial-scale=1, shrink-to-fit=no”>
```

Comment:
`“viewport”` => the quotes look weird here -- did you test it on your computer?

Reply:
Yes sorry, I copy/past it from slack 🙈